resources/home/dnanexus/eggd_artemis.py

```
@@ -0,0 +1,438 @@
+
+    if snv_path and cnv_path:
+        merge(file_data, snv_data, cnv_data)
+    elif snv_path:
+        file_data = snv_data
+    elif cnv_path:
+        file_data = cnv_data
+    else:
+        logger.debug("No paths given, exiting...")
+        exit(1)
+
+    today = datetime.datetime.now().strftime("%Y-%m-%d")
```

Ruff (0.8.0)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)